docs/source/install.rst

```diff
@@ -6,87 +6,56 @@ Installation Guide
 Recommended Environments
 ------------------------
 
-We recommend these Linux distributions.
+We recommend the following Linux distributions.
 
-* `Ubuntu <https://www.ubuntu.com/>`_ 14.04/16.04 LTS 64bit
-* `CentOS <https://www.centos.org/>`_ 7 64bit
-
-The following versions of Python can be used: 2.7.6+, 3.4.3+, 3.5.1+, and 3.6.0+.
-
-.. warning::
-
-   If you are using certain versions of conda, it may fail to build CuPy with error
-   ``g++: error: unrecognized command line option ‘-R’``.
-   This is due to a bug in conda (see `conda/conda#6030 <https://github.com/conda/conda/issues/6030>`_ for details).
-   If you encounter this problem, please downgrade or upgrade it.
+* `Ubuntu <https://www.ubuntu.com/>`_ 14.04 / 16.04 LTS (64-bit)
+* `CentOS <https://www.centos.org/>`_ 7 (64-bit)
 
 .. note::
 
-   We are testing CuPy automatically with Jenkins, where all the above *recommended* environments are tested.
+   We are automatically testing CuPy on all the recommended environments above.
    We cannot guarantee that CuPy works on other environments including Windows and macOS, even if CuPy looks running correctly.
```

Comment:
Could you change this sentence to `... even if CuPy may seem to be running correctly.` too?